src/network/networkctl.c

```diff
@@ -42,6 +42,7 @@
 #include "pager.h"
 #include "parse-util.h"
 #include "pretty-print.h"
+#include "sd-dhcp-client.h"
```

Comment:
we usually include our own public headers at the top, before our own local headers

Reply:
ok